### PR TITLE
Migrate updateAttribues, remove the class

### PR DESCRIPTION
**Pull request by JackWilb**
Depends on #149 

Another step towards #141 

I moved the remaining logic from the class to the MultiMatrix file, `updateAttributes`. I also removed the eventBus and we now update the network with an emit up to the Controls component. 

The next PR for `remove-class` should just clean up the remaining bugs, and then we can merge it back to master.

### CODE REVIEW

**Review by waxlamp — APPROVED** [review]
Nice!

**Comment by JackWilb** [comment]
Force pushed to clean up the commit history here. I didn't change the files at all

File: src/components/MultiMatrix/MultiMatrix.vue [hunk 1]
```diff
@@ -825,6 +802,172 @@ export default Vue.extend({
...
+        .on('click', (d: string) => {
+          if (this.enableGraffinity) {
+            this.$emit(
+              'updateNetwork',
+              superGraph(this.network.nodes, this.network.links, d),
+            );
+          } else {
+            this.sort(d);
+          }
+        });
```
Comment (gotdairyya) [inline src/components/MultiMatrix/MultiMatrix.vue:841]: #152

**Review by gotdairyya — COMMENTED** [review]
Looks great!

File: src/components/MultiMatrix/MultiMatrix.vue (hunk 1 above)
Comment (gotdairyya) [reply src/components/MultiMatrix/MultiMatrix.vue:841]: Moved this to issue

**Pull request merged**